src/app/features/courses/courses-list/courses-list.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="courses-list">
+  <app-course-card
+    *ngFor="let course of courses"
+    [title]="course.title"
+    [description]="course.description"
+    [creationDate]="course.creationDate"
+    [duration]="course.duration"
+    [authors]="course.authors"
+    [editable]="editable"
+    (clickOnShow)="onShowCourse('show clicked')"
+  >
+    <app-button
+      *ngIf="editable"
+      [iconName]="'trash'"
+      (click)="onEditCourse('edit clicked')"
+    ></app-button>
+    <app-button
+      *ngIf="editable"
+      [iconName]="'pen'"
+      (click)="onDeleteCourse('remove clicked')"
```

Comment:
Handlers: onShowCourse, onEditCourse, onDeleteCourse shall send course name to the parent component:

```html

   (clickOnShow)="onShowCourse(course.title)"

```